packages/fxa-content-server/app/scripts/lib/oauth-errors.js

```diff
@@ -177,6 +177,11 @@ var ERRORS = {
     message: t('Unverified user or session'),
     response_error_code: 'interaction_required',
   },
+  PROMPT_NONE_INVALID_ID_TOKEN_HINT: {
+    errno: 1013,
+    message: t('Invalid id_token_hint'),
+    response_error_code: 'invalid_request',
```

Comment:
I'm not sure if this message should be translated. Also not sure if this should be an `interaction_required` error, rather than an `invalid_request`.

Reply:
hm `invalid_request` seems fine 👁️